data/tokens.json

```diff
@@ -3904,6 +3904,77 @@
         "value": "{gtmds_shadow-md}",
         "type": "boxShadow"
       }
+    },
+    "Canvas Menu": {
+      "gtmds_canvas-menu_shadow-sm": {
+        "value": {
+          "x": "0",
+          "y": "1",
+          "blur": "1",
+          "spread": "0",
+          "color": "#5A7F9E22",
+          "type": "dropShadow"
+        },
+        "type": "boxShadow"
+      },
+      "gtmds_canvas-menu_shadow-md": {
+        "value": {
+          "x": "0",
+          "y": "2",
+          "blur": "6",
+          "spread": "0",
+          "color": "#5A7F9E22",
+          "type": "dropShadow"
+        },
+        "type": "boxShadow"
+      },
+      "gtmds_canvas-menu-item_shadow": {
+        "value": {
+          "x": "0",
+          "y": "1",
+          "blur": "2",
+          "spread": "0",
+          "color": "#5A7F9E4D",
+          "type": "innerShadow"
+        },
+        "type": "boxShadow"
+      },
+      "gtmds_canvas-menu-item_color": {
+        "value": "{gtmds_color_gray-9}",
+        "type": "color"
+      },
+      "gtmds_canvas-menu-item_color-hover": {
```

Comment:
shouldn't those be inherited by Icon Button's tokens instead of having their own?

Reply:
we are reworking the icon buttons right now but I would generally say yes, we should have as little overrides for styles as possible